python/tskit/trees.py

```diff
@@ -3673,7 +3673,13 @@ def trees(
         )
         return TreeIterator(tree)
 
-    def haplotypes(self, impute_missing_data=False, missing_data_character="-"):
+    def haplotypes(
+        self,
+        *,
+        isolated_as_missing=None,
+        missing_data_character="-",
+        impute_missing_data=None,
```

Comment:
I don't think ``impute_missing_data`` was ever released for haplotypes, so we can probably just nuke it. (best to check, though)

Reply:
From what I can see it was released: https://github.com/tskit-dev/tskit/blob/0.2.3/python/tskit/trees.py#L2830

Reply:
Ah, my bad. Thanks.